forum/communications/models.py

```diff
@@ -1,3 +1,18 @@
 from django.db import models
 
-# Create your models here.
+from mongoengine import EmbeddedDocument, StringField, Document, DateTimeField, ListField, \
+    EmbeddedDocumentField
+
+class User(EmbeddedDocument):
+    user_id: StringField(required=True)
+    username: StringField(required=True)
```

Comment:
You're using `user_id: StringField(...)` as a type annotation, which can cause issues. In MongoEngine, fields should be class attributes, not type annotations:
```
class User(EmbeddedDocument):
    user_id = StringField(required=True)
    username = StringField(required=True)
```

Reply:
I completely agree that it should be class attributes, it just typo. Thank you for your attention.